4981_ChatRoom/client.cpp

```diff
@@ -264,80 +270,132 @@ void concatIP(char *ip)
     strcat(ip, tmpIP);
 }
 
-
 /********************************************************
- *  Function:       char * getUserMessage(ClientWindow *cw, char *IP, char *username)
- *                      ClientWindow *cw: ClientWindow pointer in order to access the UI
- *                      char *completeMsg: buffer that holds the complete message to send
+ *  Function:       void formatMessage(char *message, char *IP, ClientWindow *main)
+ *                      char *message: buffer that holds message to be formatted.
  *                      char *IP: string containing the IP of the client sending the message
- *                      char *username: client username
+ *                      ClientWindow *main: The UI for the client that sent the message.
  *
- *  Programmer:     Robert Arendac
+ *  Programmer:     Matt Goerwell
  *
  *  Created:        Mar 11 2017
  *
- *  Modified:
+ *  Modified:       Mar 15 2017 - strings instead of char *
+ *                  Mar 16 2017 - Including display and send method calls
  *
  *  Desc:
- *      Will get the message entered by the user into the GUI.  Will then
- *      put a string with the IP and username appended to the front into the completeMsg buffer.
+ *      This will take the message passed into it and apply standard formatting rules.
+ *      That is to say, It will prepend the message with the timestamp, username, and
+ *      IP address of the sender. It will also colour code the username eventually.
  *******************************************************/
-void getUserMessage(ClientWindow *cw, char *completeMsg, char *IP, char *username)
+void formatMessage(const char *message, const char *IP,ClientWindow *main)
```

Comment:
formatMessage does more than you say it does, also it's doing more than just formatting.  Rename or refactor the flow a bit